.env

```diff
@@ -39,7 +39,6 @@ PLACE_REST_API_TAG=beta
 PLACE_RUBBER_SOUL_TAG=beta
 # PLACE_TRIGGERS_TAG=beta
 # PLACE_SOURCE_TAG=beta
-# PLACE_NGINX_TAG=latest
```

Comment:
This can remain as a placeholder / for overriding when required.

Reply:
yep I had thought because the env var is now set dynamically in `install` script, that we would need to remove it from `.env` to ensure that static default wouldn't override the dynamic value set in `install`.

But happily, docker-compose will prioritise shell env vars first before .env values
https://docs.docker.com/compose/environment-variables/#the-env-file 

So I'll uncomment this line

Reply:
oh actually, setting the value in .env will have no affect if .install is used,

but i've re-added the placeholder in .env anyway in case the user wants to set it and run `docker-compose up` manually